index.js

```diff
@@ -0,0 +1,111 @@
+const playerContainer = document.querySelector(".player-container")
+
+function renderPlayer(player) {
+  const playerDiv = document.createElement("div")
+  playerDiv.className = "player"
+  playerDiv.setAttribute("player-id", player.id)
```

Comment:
This will work, but it's better to use a dataset in cases like these - since "player-id" is not a standard HTML attribute. You can use this instead:

```js
div.setAttribute("data-id", player.id)
```

*or*

```js
div.dataset.id = player.id
```